person.rb

```diff
@@ -10,6 +10,7 @@ def initialize(age, parent_permission: true, name: 'unknown')
     @name = name
     @age = age
     @parent_permission = parent_permission
+    @rentals = []
   end
 
```

Comment:
- [ ] It would be great if you add the add_rental method in your person class as suggested by the previous reviewer to complete the `many-to-many` association 😊

```ruby
def add_rental(book, date)
  Rental.new(date, book, self)
end
```